Platform/ArrowlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -10,21 +10,17 @@
 #include <Library/MeExtMeasurementLib.h>
 #include "Stage2BoardInitLib.h"
 #include "PsdLib.h"
-#include "GpioPinsVer4S.h"
-#include "GpioPinsVer2Lp.h"
+#include <GpioV2PinsMtlPchS.h>
 #include <Library/TimerLib.h>
-#include <Include/PcrDefine.h>
 #include <GpioV2PinsMtlSoc.h>
 #include <Library/GpioV2Lib.h>
-#include <Include/GpioV2Config.h>
-#include <Library/MtlSocGpioTopologyLib.h>
-#include "GpioTableArlSPostMem.h"
-#include "GpioTableArlhPostMem.h"
-#include "GpioTableArlhRvpPostMem.h"
+#include <GpioV2Config.h>
+#include <Library/GpioV2SiLib.h>
 #include <Library/PciePm.h>
-#include <Library/MtlPchGpioTopologyLib.h>
 #include <Library/MtlSocInfoLib.h>
-
+#include <Library/P2SbLib.h>
+#include <Guids/FspPchConfigHob.h>
```

Comment:
@gdong1 pls commit this file. thanks

Reply:
Thanks to capture this. It was filtered by the .gitignore. will add it.